static/app/utils/performance/contexts/pageError.tsx

```diff
@@ -1,49 +1,78 @@
-import {createContext, useContext, useState} from 'react';
+import {createContext, Fragment, useContext, useRef, useState} from 'react';
 import {Theme} from '@emotion/react';
 
 import {Alert} from 'sentry/components/alert';
+import {IconClose} from 'sentry/icons';
 
-export type PageAlert = {
-  message: string;
+export type PageAlertOptions = {
+  message: string | undefined;
```

Comment:
1. Why `| undefined`? Is that the mechanism for removing an alert?
2. I think it should be `React.ReactNode` or something similar, so hook users can pass in formatted DOM elements

Reply:
1. Yeah pretty much, and also to support things like without further refactor https://github.com/getsentry/sentry/pull/61881/files#diff-406fb42bb51b542389d6d77c4d299d2c61606d9773204c1d4cc9689aac402188R70
where msg is `string | undefined`. 

2. `React.ReactNode` is a great idea, i'll make the change!